modules/ui/src/Form/StockTransactions1.php

```diff
@@ -43,14 +43,6 @@ public function buildForm(array $form, FormStateInterface $form_state) {
   /**
    * {@inheritdoc}
    */
-  public function validateForm(array &$form, FormStateInterface $form_state) {
```

Comment:
This is looks like a valid comment. We need to disprove it or replace it with an issue.
Why is this breaking tests?